momento-redis/scalar_test.go

```diff
@@ -0,0 +1,365 @@
+package momento_redis_test
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/momentohq/client-sdk-go/auth"
+	"github.com/momentohq/client-sdk-go/config"
+	"github.com/momentohq/client-sdk-go/momento"
+	momentoredis "github.com/momentohq/momento-go-redis-client/momento-redis"
+	. "github.com/momentohq/momento-go-redis-client/momento-redis/test_helpers"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redis/go-redis/v9"
+)
+
+var _ = Describe("Scalar methods", func() {
+	sContext := NewSharedContext()
+	BeforeEach(func() {
+		cacheName := fmt.Sprintf("golang-redis-%s", uuid.NewString())
+		switch sContext.UseRedis {
+		case true:
+			// change if running on different host and port
+			host := "127.0.0.1"
+			port := "6379"
+			sContext.Client = redis.NewClient(&redis.Options{
+				Addr: host + ":" + port,
+			})
+		case false:
+			credential, _ := auth.NewEnvMomentoTokenProvider(AuthTokenEnvVariable)
+			mClient, _ := momento.NewCacheClient(config.LaptopLatest(), credential, 60*time.Second)
+			sContext.MomentoClient = mClient
+			// create cache; it resumes execution normally incase the cache already exists and isn't exceptional
+			sContext.CreateCache(sContext.Ctx, mClient, cacheName)
+			sContext.Client = momentoredis.NewMomentoRedisClient(mClient, cacheName)
```

Comment:
Can you go ahead and add error handling to these setup steps please? I had a problem with my auth token environment variable but the only error being surfaced in my test output was about `invalid memory address or nil pointer dereference`.